Resources/views/CRUD/base_list.html.twig

```diff
@@ -17,6 +17,20 @@ file that was distributed with this source code.
 
 {% block tab_menu %}{{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active', 'template': sonata_admin.adminPool.getTemplate('tab_menu_template')}, 'twig') }}{% endblock %}
 
+{% block title %}
+    {#
+        The list template can be used in nested mode,
+        so we define the title correponding to the parent's admin.
+    #}
+    {% if admin.isChild and admin.parent.subject %}
+        {{ "title_edit"|trans({'%name%': admin.parent.toString(admin.parent.subject)|truncate(15) }, 'SonataAdminBundle') }}
```

Comment:
[We are going to deprecate and remove the text extension (and thus, `truncate`)](https://github.com/sonata-project/SonataCoreBundle/pull/311) :/